wren-ui/src/apollo/server/services/tests/dashboardService.test.ts

```
@@ -0,0 +1,977 @@
+  }
+
+  public testToUTC(schedule: DashboardSchedule): DashboardSchedule {
+    return this.toUTC(schedule);
+  }
+
+  public testToTimezone(schedule: DashboardSchedule): DashboardSchedule {
+    return this.toTimezone(schedule);
+  }
+
+  public testValidateScheduleInput(data: SetDashboardCacheData): void {
+    return this.validateScheduleInput(data);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)